appengine-standard-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/logging.properties

```diff
@@ -3,11 +3,43 @@
 #
 # To use this configuration, copy it into your application's WEB-INF
 # folder and add the following to your appengine-web.xml:
-# 
+#
 # <system-properties>
 #   <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
 # </system-properties>
 #
 
-# Set the default logging level for all loggers to WARNING
-.level = WARNING
+# Logging configuration file for Google App Engine tools.
+
+# Specify the handlers to create in the root logger
+# (all loggers are children of the root logger)
+# The following creates the console handler
+handlers = java.util.logging.ConsoleHandler
+
+# Set the default logging level for the root logger
+.level = INFO
+
+# Set the default logging level for the datanucleus loggers
+DataNucleus.JDO.level=WARNING
```

Comment:
does this sample use datanucleus?  the above pom.xml doesn't seem to include a datanucleus plugin.  (although it's hard to see the full paths here so maybe it's for a different archetype?)